docs/security/tokens/overview.md

```diff
@@ -193,9 +193,6 @@ These are the collaborations that support or partially support using tokens for
 | LIGO          | Yes                    |
 | OSG           | Yes                    |
 
-Until all of the collaborations you support are using tokens in production, your CE should remain on OSG 3.5,
-with the 3.5-upcoming repositories enabled.
-
```

Comment:
Let's just blow out the entire collaboration support section

Reply:
done